src/client/containers/AboutPage/AboutPage.Style.css

```diff
@@ -1,13 +1,30 @@
-.AboutpageContainer {
+.aboutContainer {
   padding: 15px;
 }
 
-.AboutpageContainer img {
+.aboutContainer img {
   width: 100%;
-  height: auto;
+  height: 100%;
 }
 
-.AboutpageContainer p {
+.aboutContainer p {
   text-indent: 50px;
   text-align: justify;
 }
```

Comment:
this block has no any effect since you don't have `p `element on your about container so you can replace `.aboutContainer p` to `.aboutText` and also i suggest to give some `line-height` and `padding-right` for the text content. but note that when you add line height, you need to manage the image height may be less to 75% instead of 100%

Reply:
Yes done :)